src/CarbonAware.WebApi/test/integrationTests/CarbonAwareControllerTests.cs

```diff
@@ -188,16 +188,19 @@ public async Task EmissionsForecastsBatch_MissingRequiredParams_ReturnsBadReques
 
     [TestCase("2021-09-01T08:30:00Z", "2021-09-01T09:00:00Z", "2021-09-01T12:00:00Z", "eastus", 1, TestName = "EmissionsForecastsBatch expects OK for eastus 1 element")]
```

Comment:
```suggestion
    [TestCase("2021-09-01T08:30:00Z", "2021-09-01T09:00:00Z", "2021-09-01T12:00:00Z", "eastus", 1, TestName = "EmissionsForecastsBatch expects OK for single element")]
```

Sounded like the actual case under test is the number of elements rather than the location, so let's call that out to be clear (if that's accurate)